<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolDialog</class>
 <widget class="QDialog" name="VolDialog">
  <property name="windowTitle"><string>Vol</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item><widget class="QLineEdit" name="lineCode"><property name="placeholderText"><string>Code vol</string></property></widget></item>
   <item><widget class="QLineEdit" name="lineDestination"><property name="placeholderText"><string>Destination</string></property></widget></item>
   <item>
    <widget class="QDateEdit" name="dateEdit">
     <property name="calendarPopup"><bool>true</bool></property>
     <property name="date"><date>2025-01-01</date></property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="time"><time>12:00:00</time></property>
    </widget>
   </item>
   <item><widget class="QLineEdit" name="lineCompagnie"><property name="placeholderText"><string>Compagnie</string></property></widget></item>
   <item>
    <widget class="QDoubleSpinBox" name="spinPrix">
     <property name="maximum"><double>100000.00</double></property>
     <property name="decimals"><number>2</number></property>
     <property name="value"><double>0.00</double></property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBox">
     <item><widget class="QPushButton" name="btnCancel"><property name="text"><string>Annuler</string></property></widget></item>
     <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Enregistrer</string></property></widget></item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>